pkg/storage/storage.go

```diff
@@ -300,6 +300,7 @@ type AssertionsBackend interface {
 type ReadChangesFilter struct {
 	ObjectType    string
 	HorizonOffset time.Duration
+	StartTime     *time.Time
```

Comment:
**thought**: Will time.Time's zero value (`January 1, year 1, 00:00:00 UTC`) consider a valid value for StartTime through the API? If not, we can possibly just drop the pointer here altogether.

Reply:
Replaced with reference